sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -237,106 +272,66 @@ public AzureMonitorExporterBuilder credential(TokenCredential credential) {
      * exporter is an implementation of OpenTelemetry {@link SpanExporter}.
      *
-     * @return An instance of {@link AzureMonitorTraceExporter}.
+     * @return An instance of {@link SpanExporter}.
      * @throws NullPointerException if the connection string is not set on this builder or if the
      * environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
      */
-    public SpanExporter buildTraceExporter() {
+    public SpanExporter buildSpanExporter() {
         ConfigProperties defaultConfig = DefaultConfigProperties.create(Collections.emptyMap());
-        internalBuildAndFreeze(defaultConfig);
-        // TODO (trask) how to pass along configuration properties?
-        return buildTraceExporter(defaultConfig);
+        return buildSpanExporter(defaultConfig);
     }
 
     /**
-     * Creates an {@link AzureMonitorMetricExporter} based on the options set in the builder. This
-     * exporter is an implementation of OpenTelemetry {@link MetricExporter}.
-     *
-     * <p>When a new {@link MetricExporter} is created, it will automatically start {@link
-     * HeartbeatExporter}.
+     * Creates an Azure Monitor span exporter based on the options set in the builder. This
+     * exporter is an implementation of OpenTelemetry {@link SpanExporter}.
      *
-     * @return An instance of {@link AzureMonitorMetricExporter}.
+     * @param configProperties The OpenTelemetry configuration properties.
+     * @return An instance of {@link SpanExporter}.
      * @throws NullPointerException if the connection string is not set on this builder or if the
      * environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
      */
-    public MetricExporter buildMetricExporter() {
-        ConfigProperties defaultConfig = DefaultConfigProperties.create(Collections.emptyMap());
-        internalBuildAndFreeze(defaultConfig);
-        // TODO (trask) how to pass along configuration properties?
-        return buildMetricExporter(defaultConfig);
+    public SpanExporter buildSpanExporter(ConfigProperties configProperties) {
+        internalBuildAndFreeze(configProperties);
+        return new AzureMonitorTraceExporter(createSpanDataMapper(configProperties), builtTelemetryItemExporter,
+            statsbeatModule);
     }
 
     /**
-     * Creates an {@link AzureMonitorLogRecordExporter} based on the options set in the builder. This
+     * Creates an Azure Monitor log record exporter based on the options set in the builder. This
      * exporter is an implementation of OpenTelemetry {@link LogRecordExporter}.
      *
-     * @return An instance of {@link AzureMonitorLogRecordExporter}.
+     * @return An instance of {@link LogRecordExporter}.
      * @throws NullPointerException if the connection string is not set on this builder or if the
      * environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
      */
     public LogRecordExporter buildLogRecordExporter() {
```

Comment:
would quarkus use the no-arg or the 1-arg version of this method? if it will use the 1-arg version, maybe it makes sense to remove the no-arg version since this is a low-level api...

Reply:
It would be the no-arg version. 

The no-arg version is built from the  1-arg version. So, I have removed the `buildLogRecordExporter()`, `buildMetricExporter()`, `buildSpanExporter()` methods.